README.rst

```diff
@@ -101,6 +101,20 @@ as long as you see ``Successfully installed`` at the end.
 .. _issue #61:
    https://github.com/IntelPNI/brainiak/issues/61
 
+
+Examples
+========
+
+We include BrainIAK usage examples in the examples directory of the code
```

Comment:
I would rather keep the README short. This is only an invitation for users to look in the examples directory.